hoomd/mpcd/SystemData.h

```diff
@@ -57,7 +57,7 @@ class PYBIND11_EXPORT SystemData
     //! Get the current global simulation box
     const BoxDim& getGlobalBox() const
```

Comment:
```suggestion
    const BoxDim getGlobalBox() const
```
Let's not return references to values managed by shared pointers.

I don't think this would cause the problem with the unit test.

Reply:
Agreed, it seems unlikely this is the issue